syntax = "proto3";

package Test;

option cc_generic_services = true;

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    int32 code = 1;
    string msg = 2;
}

service TestService {
    rpc login(LoginRequest) returns (LoginResponse);
}
